src/store.js

```diff
@@ -0,0 +1,12 @@
+import { createStore, applyMiddleware } from 'redux';
+import thunk from 'redux-thunk';
+import rootReducer from './reducers/rootReducer';
+
+const Store = createStore(
```

Comment:
Is there a reason for using a capital S?

Reply:
No, there is no particular reason. Let me change that to `store`